packages/material-ui/src/Card/Card.js

```diff
@@ -1,24 +1,57 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
+import { unstable_composeClasses as composeClasses } from '@material-ui/unstyled';
+import experimentalStyled from '../styles/experimentalStyled';
+import useThemeProps from '../styles/useThemeProps';
 import Paper from '../Paper';
-import withStyles from '../styles/withStyles';
+import { getCardUtilityClass } from './cardClasses';
 
-export const styles = {
+const overridesResolver = (props, styles) => styles.root || {};
+
+const useUtilityClasses = (styleProps) => {
+  const { classes } = styleProps;
+
+  const slots = {
+    root: ['root'],
+  };
+
+  return composeClasses(slots, getCardUtilityClass, classes);
+};
+
+const CardRoot = experimentalStyled(
+  Paper,
+  {},
+  {
+    name: 'MuiCard',
+    slot: 'Root',
+    overridesResolver,
+  },
+)(() => {
   /* Styles applied to the root element. */
-  root: {
+  return {
     overflow: 'hidden',
-  },
-};
+  };
+});
+
+const Card = React.forwardRef(function Card(inProps, ref) {
+  const props = useThemeProps({
+    props: inProps,
+    name: 'MuiCard',
+  });
+
+  const { className, raised = false, ...other } = props;
 
-const Card = React.forwardRef(function Card(props, ref) {
-  const { classes, className, raised = false, ...other } = props;
+  const styleProps = { ...props };
```

Comment:
```suggestion
  const styleProps = { ...props, raised };
```

Reply:
My bad, did pass it because it is not used in the actual component but there are variants.